SomerenUI/SomerenUI.cs

```diff
@@ -543,5 +543,51 @@ private void listViewActivitiesSupervisors_ItemSelectionChanged(object sender, L
                 ShowMessage(Properties.Resources.ErrorMessage, ex.Message);
             }
         }
+
+        private void menuItemStudents_Click_1(object sender, EventArgs e)
```

Comment:
I think a mistake was made here, probably by clicking a control accidentally. Since it's name _1, there is another method in the file with the same name

Reply:
I believe I didn't find another method and didn't want to change your code without knowing the context. Should I just remove _1 from the name?

Reply:
If you're able to remove it without anything breaking, then yes, please.